res/css/photon-components.css

```diff
@@ -121,3 +154,139 @@
 .photon-message-bar-error .photon-button:hover:active {
   background-color: var(--red-90);
 }
+
+/* Label */
```

Comment:
Optional for this PR: maybe we could split this CSS file in several files, but `@import` them all here. I think our webpack configuration would merge them together.